integration_test/functions/src/index.ts

```diff
@@ -22,17 +23,19 @@ const firebaseConfig = JSON.parse(process.env.FIREBASE_CONFIG);
 admin.initializeApp();
 
 async function callHttpsTrigger(name: string, data: any) {
-  const resp = await fetch(
-    `https://${REGION}-${firebaseConfig.projectId}.cloudfunctions.net/${name}`,
-    {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ data }),
-    }
+  const url = `https://${REGION}-${firebaseConfig.projectId}.cloudfunctions.net/${name}`;
+  const client = await new GoogleAuth().getIdTokenClient(
+    '32555940559.apps.googleusercontent.com'
```

Comment:
What is this hard-coded project-specific string?

Reply:
Doing some research, this is the CLI appID? Whatever it is, can't we assume that the default compute service account has invoker permissions and use the ADC?